SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriverGameplayDebuggerContext.cpp

```diff
@@ -219,10 +224,18 @@ void USpatialNetDriverGameplayDebuggerContext::TickServer()
 		AGameplayDebuggerCategoryReplicator* CategoryReplicator = Cast<AGameplayDebuggerCategoryReplicator>(EntityObjectWeakPtr.Get());
 		if (CategoryReplicator == nullptr)
 		{
-			// Expected - assume latency and wait for actor to activate
+			// Not unexpected - assume latency and wait for actor to appear
 			continue;
 		}
 
+		// Update tracking mode, and if changed send an update
+		const bool ShouldTrackPlayer = CategoryReplicator->GetServerTrackingMode() == EGameplayDebuggerServerTrackingMode::Player;
+		if (EntityData->Component.TrackPlayer != ShouldTrackPlayer)
+		{
+			EntityData->Component.TrackPlayer = ShouldTrackPlayer;
+			ComponentsUpdated.Add(*It);
```

Comment:
Somewhat external to this PR, but why populate component updates here, only to wait for the next frame to action? Is it worth re-ordering so that the ComponentsUpdated is parsed after the actors are iterated here?

Reply:
Will reorder.